app/src/main/java/org/mozilla/fenix/cfr/SearchWidgetCFR.kt

```diff
@@ -34,7 +33,7 @@ class SearchWidgetCFR(
 ) {
 
     fun displayIfNecessary() {
-        if (!context.settings().shouldDisplaySearchWidgetCFR() || !FeatureFlags.searchWidgetCFR) { return }
+        if (!context.settings().shouldDisplaySearchWidgetCFR() || !context.settings().isInSearchWidgetExperiment) { return }
```

Comment:
nit - can we flip these two checks because the latter is a simpler check?

Reply:
Sure, good call :)